models/BaseLucee.cfc

```diff
@@ -967,6 +969,8 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
 				return 'jdbc:postgresql://{host}:{port}/{database}';
 			case 'MSSQL' :
 				return 'jdbc:sqlserver://{host}:{port}';
+			case 'MSSQL2' :
```

Comment:
With the translation fix, this case would actually be `JTDS` since acts on the "Lucee version" of the name.